prow/plugins/lgtm/lgtm.go

```diff
@@ -60,9 +60,37 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin is not configurable.
+	willNot := func(b bool) string {
+		if b {
+			return "will"
+		}
+		return "will not"
```

Comment:
To make it consistent, please remove the extra whitespace in `prow/plugins/approve/approve.go` `return "do not "` and `return "will not "`